medcat/cat.py

```diff
@@ -899,6 +929,7 @@ def multiprocessing(self,
         if save_dir_path is not None:
             os.makedirs(save_dir_path, exist_ok=True)
 
+        # "5" looks like a magic number here so better with comment about why the choice was made.
```

Comment:
cannot explain it, magic number, happy to write that :D

Reply:
Ye, a constant with a meaningful name will help.